pkg/pillar/hypervisor/kubevirt.go

```diff
@@ -542,6 +542,12 @@ func (ctx kubevirtContext) Start(domainName string) error {
 	for {
 		_, err = virtClient.ReplicaSet(kubeapi.EVEKubeNameSpace).Create(repvmi)
 		if err != nil {
+			if errors.IsAlreadyExists(err) {
+				// VMI could have been already started, for example failover from other node.
+				// Its not an error, just proceed.
+				logrus.Infof("Start VMI replicaset failed %v\n", err)
```

Comment:
Even though this error is harmless, it should be marked at least as a `Warnf`, I see other errors below marked as `Infof`, they should be `Errorf`....

Reply:
Sure will do